src/components/DailyTasks.js

```diff
@@ -1,21 +1,19 @@
+import {useState} from 'react';
 import styled from 'styled-components';
 import TaskList from './TaskList';
-
-const dailyTasks = [
-  'Räume lüften',
-  'Betten machen',
-  'Spülmaschine ausräumen',
-  'Wäsche machen',
-  'Müll rausbringen',
-  'Böden saugen',
-  'ungenutzte Dinge wegräumen',
-  'Waschbecken im Bad reinigen',
-  'Toilette im Bad reinigen',
-  'Arbeitsflächen in der Küche abwischen',
-  'volle Spülmaschine anschalten',
-];
+import dailyTasks from '../db';
 
 export default function DailyTasks() {
+  const [todos, setTodos] = useState(dailyTasks);
+
+  function handleTodos(todoToHandle) {
+    setTodos(
+      todos.map(todo => {
+        return todo.id === todoToHandle.id ? {...todo, isChecked: !todo.isChecked} : todo;
+      })
+    );
```

Comment:
Could be even shorter as your function consists of only one line returning something:

`setTodos(todos.map(todo => (todo.id === todoToHandle.id ? {...todo, isChecked: !todo.isChecked} : todo)));`

In this case you can omit the curly braces AND the return keyword and just write the expression you want to return. However, there is nothing wrong with your code and in the beginning it's sometimes easier to read if you have an explicit return.